lib/shelley/test/integration/Main.hs

```diff
@@ -139,6 +141,8 @@ main = withUtf8Encoding $ withLogging Nothing Info $ \(_, tr) -> do
         specWithServer tr $ do
             describe "API Specifications" $ do
                 Addresses.spec @n
+                ByronMigrations.spec @n
```

Comment:
I think it's a bit too soon to enable this one on the shelley node.

Reply:
They have been enabled (those tests were part of Wallets.spec, but they were made "pending"), but since they are now extracted to separate spec, then maybe they just don't have to be enabled for shelley indeed.

Reply:
I have removed those tests from Shelley suite and also added ByronWallet.spec. In https://github.com/input-output-hk/cardano-wallet/pull/1739/commits/c8fdef4620c4b5dfcc44305f8f397ab0acc75f5a.